.github/workflows/ci.yml

```
@@ -63,71 +60,58 @@ jobs:
           password: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Check if image exists
-        id: check-image
         run: |
           EXISTS=$(docker manifest inspect ${{ env.DOCKER_DEPENDENCY_IMAGE_NAME }}:${{ env.DIR_HASH }} > /dev/null 2>&1 && echo "true" || echo "false")
```

```suggestion
          EXISTS=$(docker manifest inspect ${{ env.DOCKER_DEPENDENCY_IMAGE_NAME }}:filehash-${{ env.DIR_HASH }} > /dev/null 2>&1 && echo "true" || echo "false")
```

Isn't this missing here?

---

Ah, yes, good catch! I somehow removed it during some late refactoring